great_expectations/metrics/__init__.py

```diff
@@ -1 +1,3 @@
+from .batch.row_count import BatchRowCount
 from .column_values.between import ColumnValuesBetween
+from .metric import Metric
```

Comment:
Since it looks like in some cases users will be needing the type hint `list[Metric]`, I simplified the `Metric` import.